tests/Behat/features/archive-block-element.feature

```diff
@@ -27,7 +27,7 @@ Feature: Archive elements in the CMS
       And I see the text "Are you sure you want to send this block to the archive?" in the alert
       And I confirm the dialog
       # Sometimes behat can be quicker than the time it takes the JS to remove the row...
-      And I wait 1 second
+      And I wait 2 seconds
```

Comment:
lol

Reply:
Race conditions be racing.
Unfortunately we were losing that race :(
Tried to update it to wait for the spinner, then wait for the elements to load... but we were too fast (and it sat waiting for the spinner when it'd been and gone).

Reply:
I've deleted this in #374 anyway

Reply:
Changed required: rebase